edu.stanford.Almond.json

```diff
@@ -0,0 +1,272 @@
+{
+    "app-id" : "edu.stanford.Almond",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.gfortran-62"
+    ],
+    "command" : "/app/bin/edu.stanford.Almond",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=host",
+        "--talk-name=ca.desrt.dconf",
+        "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.login1",
+        "--talk-name=org.gtk.vfs.Daemon",
+        "--talk-name=org.gtk.vfs.mountpoint_http",
+        "--talk-name=org.gnome.Shell.Screenshot",
+        "--talk-name=org.freedesktop.secrets"
```

Comment:
Does it store passwords to need it?

Reply:
Yes - it hosts the user's credential to various accounts (eg Twitter, Facebook, Instagram, Nest...) so that the user can operate them.
In practice, the credentials are actually stored in an encrypted sqlite database on disk; libsecret is used to store the encryption key.